tests/unit/plugins/filter/test_ipaddr.py

```diff
@@ -68,15 +68,21 @@ def test_ipaddr_empty_query(self):
         self.assertEqual(ipaddr.ipaddr("192.0.2.230"), "192.0.2.230")
         self.assertEqual(ipaddr.ipaddr("192.0.2.230/30"), "192.0.2.230/30")
         self.assertEqual(ipaddr.ipaddr([]), [])
-        with pytest.raises(
-            AnsibleFilterError,
-            match="True is not a valid IP address or network",
-        ):
-            ipaddr.ipaddr(True)
-        with pytest.raises(
-            AnsibleFilterError, match="'' is not a valid IP address or network"
-        ):
-            ipaddr.ipaddr("")
+
+        self.assertEqual(ipaddr.ipaddr(True), False)
+        self.assertEqual(ipaddr.ipaddr(""), False)
```

Comment:
I might move these under the TODO just to indicate that they are the logical equivalents of the commented tests so they don't get lost when we restore the errors